### PR TITLE
Fix various merge issues

### PR DESCRIPTION
**Pull request by Ansariel**
@marchcat 
* Restore changes from 21947778baaca205615a71a97ac8f563c998fdd3 to llwindow/llwindowwin32.cpp
* Restore changes from 3758618949684641fc94b5c9478d9002706213cc to newview/llinspecttexture.cpp
* Fix apparent merge error in LLInventoryPanel::itemChanged
* Restore changes from 1eeecfa1a8bf43a8980217ce34e3b5f4458483e0 in newview/llpaneloutfitsinventory.h
* Restore changes from b9633c17e373bfe55b29228996e8473eb041466d in newview/llpaneloutfitsinventory.h & newview/llpanelwearing.cpp
* Restore changes from f660f1f0fda4d2363d351fa550b4f8818b46c2c3 in newview/llviewertexture.cpp
* Restore changes from b9633c17e373bfe55b29228996e8473eb041466d & 98f7d73d46fdc045759023eda6409e8c791f5cb2 in newview/lloutfitgallery.cpp and newview/lloutfitslist.cpp
* Replace changes from 23729442aab7130f3368d433e8a5a9dd45ff6b98 with current implementation in develop branch
* Fix more broken changes in LLViewerTexture::saveRawImage
* Restore the changes in LLMath both from develop and maint-c
* Fix all kind of other merge errors

### CODE REVIEW

**Review by marchcat — APPROVED** [review]
Thank you very much, @Ansariel!

File: indra/newview/llagent.cpp [hunk 1]
```diff
@@ -5028,7 +5028,7 @@ void LLAgent::applyExternalActionFlags(U32 outer_flags)
                 if( delta_time > FLY_TIME
                     && delta_frames > FLY_FRAMES)
                 {
-                    setFlying(TRUE);
```
Comment (marchcat) [inline indra/newview/llagent.cpp:5031]: But how could this pass the build check with the defines removed? 🙀

Comment (Ansariel) [reply indra/newview/llagent.cpp:5031]: They are not removed yet since not all of theHavok libs have been updated yet.

Comment (marchcat) [reply indra/newview/llagent.cpp:5031]: Right, but I do that manually for every local build after the merge, just to ensure those fake bools don't sneak in.

Comment (Ansariel) [reply indra/newview/llagent.cpp:5031]: But since the Havok stub lib isn't updated yet, you cannot make an OS build with the typedefs removed.

Comment (marchcat) [reply indra/newview/llagent.cpp:5031]: And that's why it's just a test on my non-OS local builds for now. It helped me catch a couple of leftovers before, but now it doesn't seem as reliable. There are probably more defines somewhere else.

Comment (Ansariel) [reply indra/newview/llagent.cpp:5031]: Depending on the platform they are defined in various external libraries, so they might be defined on various occasions in the viewer code.

**Review by marchcat — APPROVED** [review]
Thank you again for your help!

**Pull request merged**